src/commands/periscope/helpers/periscopehelper.ts

```diff
@@ -95,7 +97,11 @@ export async function getStorageInfo(
         const storageAccKeyList = await storageClient.storageAccounts.listKeys(resourceGroupName, accountName);
         const storageKey = storageAccKeyList.keys?.find((it) => it.keyName === "key1")?.value!;
 
+        // Get container name from cluster-info default behaviour was APIServerName without
+        const containerName = await extractContainerName(clusterKubeConfig!);
```

Comment:
How do you know it's safe to apply `!` to `clusterKubeconfig` - there doesn't seem to be any validation.  If you're relying on the caller not passing `undefined` then the function signature should be `string` not `string | undefined`...?